front/README.md

```diff
@@ -13,53 +13,57 @@ If you are developing the website without the external resources, e.g using dumm
 -   `yarn` to install dependencies.
 -   `yarn local` to start the next web app locally.
 
-If you click the localhost URL in the console, you should see the Next.js app running.
+If you click the localhost URL in the console, you should see the Next.js app running. If you are on a route which relies on external resources, you will encounter errors because it is not hooked up to your AWS lambdas; follow the next steps to get that set up!
```

Comment:
Is the web app not _always_ relying on external resources? I don't see how it can be independent knowing that we have a database hosted by AWS

Reply:
you can still work on UI locally with mock data, e.g. creating a /temp route and working on some UI for contributing